.github/workflows/studioConventions.yml

```diff
@@ -11,10 +11,22 @@ on:
 jobs:
   conventions:
     name: Studio Conventions
+    continue-on-error: true
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
 
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: '6.0'
```

Comment:
you need to do exactly as you do in `compile.yml` here and extract the version from the dotnet version